resources/templates/default.yml.j2

```diff
@@ -274,8 +274,6 @@ postprocesses:
 # This is where further options such as specific versions of dependencies can be set in the furture
 vars:
   allow_extensions: false
-  # tvm.make_tool: "ninja"
-  llvm.version: "14.0.0"
```

Comment:
Update to llvm 15 (Ubuntu 20.04) or 17+ (Ubuntu 22.04). Allow using system llvm?